src/patternfly/layouts/Level/examples/level-template-group.hbs

```diff
@@ -0,0 +1,18 @@
+{{#> level}}
+	{{#> level-group}}
+		{{#> level-item}}
+			content
+		{{/ level-item}}
+		{{#> level-item}}
+			content
+		{{/ level-item}}		
+	{{/ level-group}}
+	{{#> level-group}}
+		{{#> level-item}}
```

Comment:
Please use two spaces in your ide instead of tabs for the spacing (this will have be adjusted in all of your hbs files)